extensions/podman/src/vfkit-helper.ts

```diff
@@ -0,0 +1,31 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import * as extensionApi from '@podman-desktop/api';
+
+export class VfkitHelper {
```

Comment:
Those helper all looks the same, can this be generify ?

Reply:
I think there is a little difference between qemu/krunkit and vfkit, as the first ones are installed by podman, and the other one is installed by the user. And this makes some difference for the helpers. But I would like to have confirmation before to make it generic